pkg/networkservice/connectioncontext/ipcontext/vl3/metadata.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2022 Cisco and/or its affiliates.
+// Copyright (c) 2022-2024 Cisco and/or its affiliates.
```

Comment:
unnecessary change?

Reply:
this is because I changed the name of this file